uint32 uid
uint32 system_time
uint8 io_status
IotFrame0Node[] nodes
